src/u2f/u2f_app.c

```diff
@@ -167,7 +167,7 @@ void u2f_app_confirm_start(enum u2f_app_confirm_t type, const uint8_t* app_id)
         }
         break;
     case U2F_APP_AUTHENTICATE:
-        title = "U2F authenticate";
+        title = "U2F login";
```

Comment:
Or "confirm" or just "auth" maybe? For example, a stock exchange platform where it asks users to also confirm using a U2F device in order to complete an action.

Reply:
Makes sense. Changed to `U2F auth`.

Reply:
:+1: